engine/src/main/resources/assets/shaders/highp_frag.glsl

```diff
@@ -28,10 +28,13 @@ void main() {
     //    discard;
     //}
 
-    vec3 brightColor = max(color.rgb - vec3(highPassThreshold), vec3(0.0));
-    float bright = dot(brightColor, vec3(1.0));
+   // vec3 brightColor = max(color.rgb - vec3(highPassThreshold), vec3(0.0));
+    float bright = dot(color.rgb, vec3(0.2126, 0.7152, 0.0722));;
```

Comment:
to be honest with you, i'm not actually sure. I was looking around at how other people implement a bloom high pass filter. here is a comparison of one with and without this change. I was testing to see what effects some of these shader had by letting them pass through the color and i found this one to add quite a lot of extra bloom to the scene. It would probably be better if I set up a way to view FBO's and examine the different stages. 



![snapshot21](https://cloud.githubusercontent.com/assets/854359/13910451/9d69645c-eedf-11e5-8c2b-3c23aee6284c.png)

![snapshot22](https://cloud.githubusercontent.com/assets/854359/13910469/df8c96d8-eedf-11e5-8d83-ad105ee9f636.png)